pkg/repository/project.go

```diff
@@ -342,10 +342,6 @@ func (r *projectRepository) GetProjectsAndTenants(ctx context.Context, userId st
 		tenantRoles[t.Meta.GetId()] = tenantRole
 	}
 
-	if defaultTenant == nil {
-		return nil, errorutil.Internal("unable to find a default tenant for user: %s", userId)
-	}
```

Comment:
I think this should stay? After ensuring the tenant on login there must be a default tenant present.